test.js

```diff
@@ -44,22 +44,33 @@ test('external Redis conn', function (t) {
   })
 })
 
-abs({
-  test: test,
-  buildEmitter: function () {
-    var emitter = mqemitterRedis()
-
-    emitter.subConn.on('connect', unref)
-    emitter.pubConn.on('connect', unref)
-
-    return emitter
-  },
-  persistence: function () {
-    db.flushall()
-    return persistence()
-  },
-  waitForReady: true
-})
+{
```

Comment:
This additional block is introduced to reduce the visibility of `emittersBuiltForForAbs` and avoid making it global.